src/RunCommand.php

```diff
@@ -46,22 +46,25 @@ protected function execute(InputInterface $input, OutputInterface $output)
             });
             $progress->finish();
             $vcards = array_merge($vcards, $xcards);
-            error_log(sprintf("\nDownloaded %d vCard(s)", count($vcards)));
+            $quantity = count($vcards);
+            error_log(sprintf("\nDownloaded %d vCard(s)", $quantity));
         }
 
-        // parse and convert
-        error_log("Parsing vcards");
-        $cards = parse($vcards);
-
-        // conversion
+        // classyfication
+        error_log("Dissolving groups (e.g. iCloud)");
+        $cards = classify($vcards);
+        $passel = count($cards);
```

Comment:
rename passel?

Reply:
Was immer du willst - "pfrizelpfuempf" ? Ne, im Ernst ist nur eine Variablenbezeichnung, wenn Dir etwas besser gutiert - dann immer her damit

Reply:
Mir egal, aber passel versteh ich nicht. Was ist in der Variable? Eine Anzahl von Karten? Also count?

Reply:
passel = Schar/Menge
Eine counter variable
Heißt jetzt $zcards